pyccl/halos/profiles/profile_base.py

```diff
@@ -280,29 +300,37 @@ def convergence(self, cosmo, r, M, *, a_lens, a_source):
     @mass_def_api
     @warn_api
     def shear(self, cosmo, r, M, *, a_lens, a_source):
-        """ Returns the shear (tangential) as a function of cosmology,
+        """
+        shear(cosmo, r, M, *, a_lens, a_source)
+        Returns the shear (tangential) as a function of cosmology,
         radius, halo mass and the scale factors of the
         source and the lens.
 
         .. math::
            \\gamma(R) = \\frac{\\Delta\\Sigma(R)}{\\Sigma_{\\mathrm{crit}}} =
            \\frac{\\overline{\\Sigma}(< R) -
-           \\Sigma(R)}{\\Sigma_{\\mathrm{crit}}},\\
+           \\Sigma(R)}{\\Sigma_{\\mathrm{crit}}},
 
-        where :math:`\\overline{\\Sigma}(< R)` is the average surface density
-        within R.
+        where :math:`\\overline{\\Sigma}(< R)` is cumulative surface density
```

Comment:
Same as in `cumul2d`. Is `Sigma(<R)` correct or is it missing something before the `<`?